yt/fields/field_detector.py

```diff
@@ -126,9 +126,11 @@ def __missing__(self, item):
         elif finfo is not None and finfo.particle_type:
             if "particle_position" in (item, item[1]) or \
                "particle_velocity" in (item, item[1]) or \
+               "particle_magnetic_field" in (item, item[1]) or \
                "Velocity" in (item, item[1]) or \
                "Velocities" in (item, item[1]) or \
-               "Coordinates" in (item, item[1]):
+               "Coordinates" in (item, item[1]) or \
+               "MagneticField" in ((item, item[1])):
```

Comment:
Do you need to have the extra set of ()?

Reply:
That's a typo. Nice catch!